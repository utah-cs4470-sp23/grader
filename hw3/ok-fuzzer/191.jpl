write image E to ""
type M = float
let a = 180
type T4J_ = bool
write image false to "@n"
write image false to "U"
show false
type v = W9I.
read image "t" to m
