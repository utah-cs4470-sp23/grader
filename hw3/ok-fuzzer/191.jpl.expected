(WriteCmd (VarExpr E) "")
(TypeCmd M (FloatType))
(LetCmd (ArgLValue (VarArgument a)) (IntExpr 180))
(TypeCmd T4J_ (BoolType))
(WriteCmd (FalseExpr) "@n")
(WriteCmd (FalseExpr) "U")
(ShowCmd (FalseExpr))
(TypeCmd v (VarType W9I.))
(ReadCmd "t" (VarArgument m))
